client/src/components/PredictionPanel.vue

```diff
@@ -116,21 +123,55 @@ export default defineComponent({
     },
   },
   methods: {
-    formatPercentage(number: number | undefined): string | null {
-      if (number == null || number == 0) {
+    formatPercentage(prediction: number | undefined): string | null {
+      if (prediction == null || prediction == 0) {
         return null;
       }
-      return Math.round(number * 100).toString();
+      return Math.round(prediction * 100).toString();
+    },
+    formatPredictionAsLikelihood(prediction: number | undefined): string | null {
+      if (prediction == null || prediction == 0) {
+        return null;
+      }
+      switch (true) {
+        case prediction < 0.33:
```

Comment:
can we make these values constants?

Reply:
I don't know why I didn't do this earlier...